### PR TITLE
Interaction states color picker - RSC-1464

### PR DESCRIPTION
**Pull request by EmilyBermudez88**
https://issues.sonatype.org/browse/RSC-1464

### CODE REVIEW

File: lib/src/components/NxColorPicker/NxColorPicker.scss [hunk 1]
```diff
@@ -43,14 +39,15 @@
   }
 
   &__color {
-    stroke: none;
     fill: var(--nx-selectable-color-dark);
+    stroke: var(--nx-selectable-color-dark);
+    stroke-width: 1px;
   }
-
+
```
Comment (entroform) [inline lib/src/components/NxColorPicker/NxColorPicker.scss:46]:
```suggestion
 
```

**Comment by entroform** [comment]
It looks like the color circle diameter grew by one pixel: `25px` instead of `24px` in the mockup

**Comment by EmilyBermudez88** [comment]
> It looks like the color circle diameter grew by one pixel: `25px` instead of `24px` in the mockup

Ok, that is what I suspected but was having trouble telling. My thought would be to shrink the radius of both `<circle>` elements to compensate for the stroke being added now. What do you think?

**Comment by entroform** [comment]
> > It looks like the color circle diameter grew by one pixel: `25px` instead of `24px` in the mockup
> 
> Ok, that is what I suspected but was having trouble telling. My thought would be to shrink the radius of both `<circle>` elements to compensate for the stroke being added now. What do you think?

Hmm yeah, it looks like that's the way (changing the radius)!

**Comment by EmilyBermudez88** [comment]
> > > It looks like the color circle diameter grew by one pixel: `25px` instead of `24px` in the mockup
> > 
> > 
> > Ok, that is what I suspected but was having trouble telling. My thought would be to shrink the radius of both `<circle>` elements to compensate for the stroke being added now. What do you think?
> 
> Hmm yeah, it looks like that's the way (changing the radius)!

Fixed here, thanks for catching it: [6eab0ba](https://github.com/sonatype/sonatype-react-shared-components/pull/998/commits/6eab0ba8c29b00cb534f8f37a46fe76ffc3422b1)

File: lib/src/components/NxColorPicker/NxColorPicker.scss (hunk 1 above)
Comment (EmilyBermudez88) [reply lib/src/components/NxColorPicker/NxColorPicker.scss]: fixed in this commit: [6eab0ba](https://github.com/sonatype/sonatype-react-shared-components/pull/998/commits/6eab0ba8c29b00cb534f8f37a46fe76ffc3422b1)

**Comment by entroform** [comment]
> > > > It looks like the color circle diameter grew by one pixel: `25px` instead of `24px` in the mockup
> > > 
> > > 
> > > Ok, that is what I suspected but was having trouble telling. My thought would be to shrink the radius of both `<circle>` elements to compensate for the stroke being added now. What do you think?
> > 
> > 
> > Hmm yeah, it looks like that's the way (changing the radius)!
> 
> Fixed here, thanks for catching it: [6eab0ba](https://github.com/sonatype/sonatype-react-shared-components/pull/998/commits/6eab0ba8c29b00cb534f8f37a46fe76ffc3422b1)

Thanks Emily, that looks good! just need to update the snapshots again i think

**Review by entroform — APPROVED** [review]
LGTM! Just need to update the snapshots 🚀

**Comment by EmilyBermudez88** [comment]
> LGTM! Just need to update the snapshots 🚀

Thanks- I could have sworn I pushed them up but clearly didn't! [5d50588](https://github.com/sonatype/sonatype-react-shared-components/pull/998/commits/5d50588e6c79c83e996e39b544c4e1f35720e851)

**Pull request merged**